x-pack/legacy/plugins/ml/public/application/util/url_state.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getUrlState } from './url_state';
+
+describe('getUrlState', () => {
+  test('properly docode url with _g and _a', () => {
```

Comment:
Typo! Should be `decode`

Reply:
https://github.com/elastic/kibana/pull/54915/commits/40d9646682e48e1f699c04874bed9076b1d66203